iac-extensions/cloudformation/src/main/java/org/sonar/iac/cloudformation/checks/PrivilegeEscalationCheck.java

```diff
@@ -55,7 +58,16 @@ protected void checkResource(CheckContext ctx, Resource resource) {
 
   private static void checkPrivilegeEscalation(CheckContext ctx, Policy policy, Resource resource) {
     for (Statement statement : policy.statement()) {
-      Optional<PrivilegeEscalationVector> vectorOpt = getStatementEscalationVector(statement);
+      List<Tree> actionTrees = Collections.emptyList();
```

Comment:
See the suggestion on this method in the Terraform check. If you decide to go for it there, I suggest doing it here too.